### ISSUE TITLE
Media Capabilities

### ISSUE THREAD

**Issue by padenot** [mozilla/standards-positions#184]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Media Capabilities
* Specification or proposal URL: https://wicg.github.io/media-capabilities/
* Caniuse.com URL (optional): Not yet listed, but https://github.com/Fyrd/caniuse/issues/4480 exists
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1409664
* Mozillians who can provide input (optional): @jyavenard

### Other information

Firefox has shipped this for some time and we're following new spec developments.

**Comment by marcoscaceres** [comment]
So I guess this is "important" given that we are shipping, or are we more on the "worth prototyping" territory?

**Comment by padenot** [comment]
Clearly "important", this is being used in the wild by major websites (I know YouTube uses it).

**Comment by Ccombs1993** [comment]
> ## Request for Mozilla Position on an Emerging Web Specification
> * Specification Title: Media Capabilities
> * Specification or proposal URL: https://wicg.github.io/media-capabilities/
> * Caniuse.com URL (optional): Not yet listed, but [Fyrd/caniuse#4480](https://github.com/Fyrd/caniuse/issues/4480) exists
> * Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1409664
> * Mozillians who can provide input (optional): @jyavenard
> 
> ### Other information
> Firefox has shipped this for some time and we're following new spec developments.

**Comment by annevk** [comment]
@padenot "important" is probably a bit strong given what we usually mark as such, but "worth prototyping" seems reasonable. Are you interested in creating a PR to close this?

**Comment by padenot** [comment]
We can't say it's `"worth prototyping"` because we've shipped it for a long time, it's central to media playback quality on the web, and we pay close attention to the standard and participate in its development. Maybe a label is missing?

In any case I'll certainly write the PR for this.

**Comment by annevk** [comment]
Yeah or maybe we need to rename "worth prototyping" to "supportive" or some such. (There are other entries marked as worth prototyping that we have shipped for a long time.)
